scripts/fetch_ext_bins.sh

```diff
@@ -26,7 +26,7 @@ if [[ -n "${TRACE}" ]]; then
   set -x
 fi
 
-k8s_version=1.14.1
+k8s_version=1.16.7
```

Comment:
1.16.4 should work here. It is what is currently in use by CAPI and a WIP PR for CAPA